load-test/single_msg_load.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/sh
+
+set -e
+
+# get absolute parent directory path of current file
+CURPATH=`dirname $(realpath "$0")`
+cd $CURPATH
+
+# set environment with env config.
+set -a
+source ../env
+set +a
+
+# set pythonpath environment
+cd ..
+export PYTHONPATH=$PWD:$PWD/python-qa-tools
+echo $PYTHONPATH
+
+# check environment variables are set
+bash ./deps/env-check.sh $CURPATH
```

Comment:
we are repeating this. May be we can just have a single file and execute from each script when required